docs.mk

```diff
@@ -5,4 +5,6 @@ docs: bin/plantuml.jar
 
 bin/plantuml.jar:
 	mkdir -p bin
-	curl -Lo $@ https://sourceforge.net/projects/plantuml/files/1.2022.2/plantuml.1.2022.2.jar/download
\ No newline at end of file
+	# Seems that SF is showing updated one, but no download of the previous one we were using
+	# curl -Lo $@ https://sourceforge.net/projects/plantuml/files/1.2022.2/plantuml.1.2022.2.jar/download
```

Comment:
Can we remove that line or at least add a `TODO` for it?

Reply:
done